wire-ios/Wire-iOS/Sources/UserInterface/Conversation Options/ConversationGuestOptionsViewModel.swift

```diff
@@ -64,6 +63,12 @@ final class ConversationGuestOptionsViewModel {
 
     private var link: String?
 
+    var securedLink: String? {
```

Comment:
should be `private`?

Reply:
Nope, because it's needed to be used in my VC:

```swift
private func handleGuestLinkNotification(_ notification: Notification) {
        if let link = notification.userInfo?["link"] as? String {
            viewModel.securedLink = link
        }
    }
```